src/components/button/button.jsx

```diff
@@ -0,0 +1,33 @@
+// Copied from scratch-gui for now
```

Comment:
Let's keep these comments the same so we can find all these files easily later

Right now in some of the other files the comment is:
/* DO NOT EDIT
@todo This file is copied from GUI and should be pulled out into a shared library.
See https://github.com/LLK/scratch-paint/issues/13 */

Reply:
awesome, thanks!